chatwoot/chatwoot
# feat: Toggle to disallow users to send messages after a conversation is resolved

## Description

This change adds a toggle to disallow the users to send messages after a conversation is resolved. This can be configured in the website channel settings. 

Fixes #2827

## Type of change


## How Has This Been Tested?

1. Create a new website inbox.
2. Open web widget using the 'Messenger Script' from the website inbox configuration.
3. Send a message from web widget.
4. Conversation should be created.
5. Resolve the conversation.
6. **Start a new conversation** button should be shown in webwidget. 
    a) if **Allow messages after conversation resolved** is `Disabled` in website inbox settings.
7. **Chat Input Box** should be shown in webwidget
    a) if **Allow messages after conversation resolved** is `Enabled` in website inbox settings.

## Checklist:
